mod.conf

```diff
@@ -1,5 +1,6 @@
 name = pipeworks
 description = This mod uses mesh nodes and nodeboxes to supply a complete set of 3D pipes and tubes, along with devices that work with them.
-depends = default, basic_materials, screwdriver
-optional_depends = mesecons, mesecons_mvps, digilines, signs_lib, unified_inventory
+#depends = default, basic_materials, screwdriver
```

Comment:
```suggestion
```

Reply:
ah right. forgot to remove this, thanks

Reply:
https://github.com/mt-mods/pipeworks/pull/16/commits/58bd68f79d3061f66b0df679c35ff6594ba56710 resolved